src/components/Breakdown.jsx

```diff
@@ -35,24 +35,34 @@ Breakdown.propTypes = {
     locale: PropTypes.string,
 };
 
-const BreakdownItem = ({ children, info, methodIcon, secondary, value, className, color = "default", ...rest }) => {
-    /** When BreakdownItem is directly used without outer <Breakdown /> component, the context would be `undefined` */
+const BreakdownItem = ({
+    info,
+    methodIcon,
+    secondary,
+    value,
+    color = "default",
+    className,
+    classNames = { key: "", children: "", info: "", value: "" },
```

Comment:
Adding this prop, and using it the tags below is the customization which we've long wanted.